src/HDF5.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="HDF5"
+   name="HDF5 License" listVersionAdded="3.27.0">
+      <crossRefs>
+         <crossRef>https://github.com/HDFGroup/hdf5/?tab=License-1-ov-file#readme</crossRef>
+      </crossRefs>
+      <notes>
+      </notes>
+      <text>
+         <titleText>
+            Copyright Notice and License Terms for HDF5 (Hierarchical Data Format 5) Software Library and Utilities -----------------------------------------------------------------------------

```

Comment:
I'm going to suggest the line of hyphens be put on its own line, with `<br /> before it. That's both so that it renders on its own line on the license list website, and also with an eye towards [the matching guideline on separators](https://github.com/spdx/license-list-XML/blob/main/DOCS/license-matching-guidelines-and-templates.md#6-code-comment-indicators-or-separators-).